packages/client/e2e/profile/test-a.spec.ts

```diff
@@ -0,0 +1,445 @@
+        const isMatchingRoomVisible = await matchingRoom.isVisible();
+        if (isMatchingRoomVisible === false) {
+            hasFoundRoom = false;
+
+            continue;
+        }
+
+        hasFoundRoom = true;
+    }
+    assertIsNotUndefined(matchingRoom);
+
+    await expect(matchingRoom).toBeVisible();
+
+    await matchingRoom.click();
+
+    const expectedListenersCounterAriaLabel = `2 Listeners`;
+    await expect(
+        joinerPage.locator(
+            `text="${expectedListenersCounterAriaLabel}" >> visible=true`,
+        ),
+    ).toBeVisible();
+}
+
+async function userGoesToTheMtvUsersListFromFullscreenPlayer({
+    page,
+    usersLength,
+}: {
+    page: Page;
+    usersLength: number;
+}) {
+    const listenersCounter = page.locator(`text="${usersLength} Listeners"`);
+    await expect(listenersCounter).toBeVisible();
+    await expect(listenersCounter).toBeEnabled();
+
+    await listenersCounter.click();
+
+    await expect(page.locator('text="Users list"').last()).toBeVisible();
+}
+
+async function openUserProfileFromMtvUsersList({
+    page,
+    userNickname,
+    userID,
+}: {
+    page: Page;
+    userNickname: string;
+    userID: string;
+}) {
+    const userCard = page
+        .locator(`css=[data-testid="${userNickname}-user-card"]`)
+        .last();
+    await expect(userCard).toBeVisible({
+        timeout: 30000,
+    });
+
+    await userCard.click();
+
+    await expect(
+        page
+            .locator(`css=[data-testid="${userID}-profile-page-screen"]`)
+            .last(),
+    ).toBeVisible();
+}
+
+async function minimizeMusicPlayer({ page }: { page: Page }) {
+    const hideFullscreenButton = page.locator(
+        'css=[aria-label="Minimize the music player"]',
+    );
+    await expect(hideFullscreenButton).toBeVisible();
+    await expect(hideFullscreenButton).toBeEnabled();
+    await hideFullscreenButton.click();
+}
+
+/**
+ * test-A/
+ * UserA opens the app
+ * UserB opens the app
+ * UserA creates MTV room
+ * UserB joins the UserA mtv room via the Mtv room search engine
+ * UserB goes to the Mtv room users list, and hit the UserA user card, he should see the UserA profile, following, followers, playlists counters and follow button.
+ * UserA goes on home
+ * UserA goes to his profile
+ * UserA goes to his settings
+ * UserA edit his nickname
+ * UserA change his relations visibility to follower-only, and his playlists visibility to private
+ * UserB goes back and hit the UserA card again, he should now see the new UserA nickname and should not be able to see his follower & following & playlists counters information
+ * UserB follows the UserA, he should now see the unfollow button and should be able to retrieve UserA relations profile information
+ */
+
+test('Profile test-a', async ({ browser }) => {
+    //This test is not protected from black box testing
+    //To avoid conflicts with previous tests, ftm workaround is to user specific profile group e2e tests userIndex 2 & 3
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Is it still relevant?